datafusion/functions-nested/src/array_has.rs

```diff
@@ -214,8 +214,7 @@ fn array_has_dispatch_for_array<O: OffsetSizeTrait>(
         let is_nested = arr.data_type().is_nested();
         let needle_row = Scalar::new(needle.slice(i, 1));
         let eq_array = compare_with_eq(&arr, &needle_row, is_nested)?;
-        let is_contained = eq_array.true_count() > 0;
-        boolean_builder.append_value(is_contained)
+        boolean_builder.append_value(eq_array.true_count() > 0);
```

Comment:
this is not a behavior change I don't think - just a refactor

Reply:
Yeah this was just matching the style of the other